src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,9 +1,5 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. It's good practice to use private access modifiers and provide getters and setters for accessing these fields.